pyproject.toml

```diff
@@ -2,16 +2,16 @@
 name = "arcaflow-plugin-kubeconfig"
 version = "0.1.0"
 description = ""
-authors = ["Harshith-umesh <harshith.umesh.nat@gmail.com>"]
-license = "Apache-2.0+GPL-2.0-only"
+authors = ["Arcalot Contributors"]
+license = "Apache-2.0"
 readme = "README.md"
 
 
 [tool.poetry.dependencies]
 python = "^3.9"
-arcaflow-plugin-sdk = "0.9.0"
+arcaflow-plugin-sdk = { git = "https://github.com/arcalot/arcaflow-plugin-sdk-python.git", branch="allow-scope-in-one-of" }
```

Comment:
Why are we not first merging the `allow-scope-in-one-of` branch to `main` in the `arcaflow-plugin-sdk-python` respository first?

Reply:
Clearly we don't want to merge this PR until we are ready to replace this with a version number. We should convert this to a draft for now.